ui/widgets/nuxeo-dialog.js

```diff
@@ -101,7 +101,8 @@ IronOverlayManager._overlayWithBackdrop = function() {
 
     _opened(e) {
       const isIOS = /iPhone|iPad|iPod/.test(window.navigator.userAgent);
-      if ((this.reparent && e.target.withBackdrop) || isIOS) {
+      // WEBUI-22: we need to check window.innerWidth to make sure dialog is reparented on smaller screens
+      if ((this.reparent && e.target.withBackdrop) || isIOS || window.innerWidth <= 720) {
```

Comment:
Not a big fan of hardcoding this 720 this element, since this is an application detail. If we don't find a better fix, we should consider backporting https://github.com/nuxeo/nuxeo-web-ui/commit/1eaad8eb7ef58246b7568356e3e5ec9233db9183 to fix the issue on desktop. Just not sure if this is enough for the client or not, though.

Reply:
I'm also not sure if this won't have any side effects, since this affects reparenting of `nuxeo-dialog`, which is a commonly used element.

Reply:
What about if we simply just always enabled reparenting for the nuxeo-dialog used inside nuxeo-data-table and made the overlay take 100vw and 100vh of width/height, similar to what we have above? At least we'd keep the hack a level above and centralized in a single element. I could go with this or with backporting nuxeo/nuxeo-web-ui@1eaad8e.